dotnet-core/plan.sh

```diff
@@ -25,34 +24,35 @@ pkg_deps=(
 pkg_build_deps=(
   core/patchelf
 )
-pkg_bin_dirs=(bin)
+pkg_bin_dirs=(.)
 
 do_unpack() {
   # Extract into $pkg_dirname instead of straight into $HAB_CACHE_SRC_PATH.
-  mkdir -p "$pkg_dirname"
-  tar xfz "$pkg_filename" -C "$pkg_dirname"
+  mkdir -p "$HAB_CACHE_SRC_PATH/$pkg_dirname"
+  tar xfz "$HAB_CACHE_SRC_PATH/$pkg_filename" -C "$HAB_CACHE_SRC_PATH/$pkg_dirname"
 }
 
 do_prepare() {
-  patchelf --interpreter "$(pkg_path_for glibc)/lib/ld-linux-x86-64.so.2" \
-    --set-rpath "$LD_RUN_PATH" \
-    ./dotnet
+  hab pkg install core/icu/52.1
```

Comment:
Had to do this to avoid a conflict with `core/icu`'s gcc-libs. I think rebuilding core/icu will fix that and then I can just include it as a normal dep.